src/sentry/api/serializers/models/event.py

```diff
@@ -419,6 +419,7 @@ def get_attrs(
         results = super().get_attrs(item_list, user, is_public)
         # XXX: Collapse hashes to one hash per group for now. Performance issues currently only have
         # a single hash, so this will work fine for the moment
+        # Assume that we don't need these once perf issues are generic? Check with dameli
```

Comment:
yeah we won't have perf problem on the transaction event anymore right?

Reply:
Right, not sure if this should be replaced with something else? Or does the frontend just know how to use the occurrence to replace this?

Reply:
this has been working locally. we'd just get `occurrence` with `event_data` instead of `perfProblem` from the `/latest` endpoint

Reply:
not sure if this is the same thing

Reply:
Yeah I think it's probably the same, just wanted to be sure